.tmp-slack-channels-for-review.md

```diff
@@ -1,21 +1,24 @@
 Slack channels
 --------------
 
-- `contrib-barcelona`
-- `contrib-gouvfr`: OpenFisca as a platform — Track on https://waffle.io/openfisca/openfisca-core
-- `contrib-italia`
-- `contrib-nz_aotearoa`: :flag-nz: :flag-mi:
-- `general`*: This channel is for team-wide communication and announcements. All team members are in this channel.
-- `knowledge_base`
-- `media-and-pr`: Tweets, twitter mentions and other notifications to nurture the OpenFisca community
-- `of-core`: Technical discussions
-- `of-france`: Discussions relatives à openfisca-france ou au système socio-fiscal français.
-- `of-legislation_explrr`
-- `of-ops`
-- `of-paris`
-- `of-tunisia`
-- `of-web_api`
-- `random`*: A place for non-work related flimflam […] you'd prefer to keep out of more work-related channels.
-- `tmp-coworking-msa`
+- `country-catalonia`: :flag-cat: Catalan tax and benefit system.
```

Comment:
`country-barcelona` ? `country-spain` ?

On the one hand, the model has been made for Barcelona, by the town council, and doesn't aim (yet) at modeling the whole region, nor the whole country. On the other hand, Barcelona is not a _country_ (but neither is, so far, Catalonia).

How should we deal with tax and benefit systems built by local authority ? I see several approaches:

- Always assume that they are part of a bigger country-wide model that may appear later (`country-spain`)
- call "country" any local authority that develops their own independent model (`country-barcelona`)
- find a better terminology to include local government

Reply:
Thanks a lot for the input @fpagnoux, the additional rule made explicit thanks to it makes a lot of sense, and the discussion started in https://github.com/jvalduvieco/openfisca-barcelona/issues/3 finally gives an opportunity to clear some uncertainty I have been having for a while now 😃